matrix_client/client.py

```diff
@@ -185,7 +192,11 @@ def send_emote(self, text):
     # See http://matrix.org/docs/spec/r0.0.1/client_server.html#m-image for the
     # imageinfo args.
     def send_image(self, url, name, **imageinfo):
-        return self.client.api.send_content(self.room_id, url, name, "image", imageinfo)
+        return self.client.api.send_content(self.room_id,
+                                            url,
+                                            name,
+                                            "m.image",
+                                            imageinfo)
```

Comment:
Generally, code style wise, we use the continuation style of:

```python
return self.client.api.send_content(
   self.room_id, url, name, "m.image", imageinfo,
)
```

and putting each keyword argument on a new line, e.g.:

```python
return self.client.api.send_content(
   self.room_id, url, name, "m.image",
   extra_information=imageinfo,
)
```